utils/near-performance-metrics/src/framed_write.rs

```diff
@@ -0,0 +1,369 @@
+use std::cell::RefCell;
+use std::marker::PhantomData;
+use std::ops::DerefMut;
+use std::pin::Pin;
+use std::rc::Rc;
+use std::task::{Context, Poll};
+use std::{io, task};
+
+use actix::{Actor, ActorContext, ActorFuture, AsyncContext, Running, SpawnHandle};
+use bitflags::bitflags;
+use bytes::BytesMut;
+use tokio::io::{AsyncWrite, AsyncWriteExt};
+use tokio_util::codec::Encoder;
+
+// This file was copied from actix-0.11.0-beta.1/src/io.rs
```

Comment:
Please copy their license and prepend it to this file. We will eventually extract all of that and put it into some `LICENSES` file on the top level, but for now let's do this in the individual files.

Reply:
I added it